packages/web/src/components/range/RatingsFilter.js

```diff
@@ -129,23 +124,10 @@ class RatingsFilter extends Component {
 	static defaultQuery = (value, props, includeNullValues = false) => {
```

Comment:
@ShahAnuj2610 Why `includeNullValues` is the third argument?
We can access it from `props`.

Reply:
no, the third argument comes from the `includeUnrated` in the data object
`
			const handleUpdates = () => {
				this.updateQuery(value, props, includeUnrated);
				this.locked = false;
				if (props.onValueChange) props.onValueChange(value);
			}
`

Reply:
https://github.com/appbaseio/reactivesearch/pull/1003

Reply:
But why you named it `includeNullValues`?